lib/pharos/addons/ingress_nginx.rb

```diff
@@ -34,11 +34,21 @@ def image_name
         end
       end
 
+      # ~One replica per 10 workers, min 2
+      # @return [Integer]
+      def replicas
```

Comment:
Naming is a bit misleading.. this is default backend replicas, right?

Reply:
yes, I'll rename it

Reply:
fixed